mmv1/third_party/terraform/tests/resource_notebooks_instance_test.go.erb

```diff
@@ -47,25 +46,25 @@ func TestAccNotebooksInstance_update(t *testing.T) {
 				ResourceName:            "google_notebooks_instance.instance",
 				ImportState:             true,
 				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"vm_image"},
+				ImportStateVerifyIgnore: []string{"vm_image", "metadata"},
 			},
 			{
-				Config: testAccNotebooksInstance_update(context),
+				Config: testAccNotebooksInstance_update(context, true),
 			},
 			{
 				ResourceName:            "google_notebooks_instance.instance",
 				ImportState:             true,
 				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"vm_image"},
+				ImportStateVerifyIgnore: []string{"vm_image", "metadata"},
 			},
 			{
-				Config: testAccNotebooksInstance_basic(context),
+				Config: testAccNotebooksInstance_update(context, false),
 			},
 			{
 				ResourceName:            "google_notebooks_instance.instance",
 				ImportState:             true,
 				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"vm_image"},
+				ImportStateVerifyIgnore: []string{"vm_image", "metadata"},
```

Comment:
Am I understanding this correctly that we aren't ever checking that the metadata is updated on the server?

Reply:
yes. The field is marked as `ignore_read` on the resource, so we never check what the value on the server is. I had to add it here because I introduced it as a new field in this test.

I suspect that we would not use ignore_read like this if implementing the resource today - but fixing that wouldn't be trivial and is outside of the scope of the ticket. In the future I think we'd want to do something similar to https://github.com/hashicorp/terraform-provider-google/issues/14293.